src/app/features/tech-record/components/tech-record-summary/tech-record-summary.component.ts

```diff
@@ -129,20 +126,17 @@ export class TechRecordSummaryComponent implements OnInit {
       /*  4 */ getTypeApprovalSection(VehicleTypes.HGV),
       /*  5 */ ApplicantDetails,
       /*  6 */ DocumentsTemplate,
-      /*  7 */ getBodySection(VehicleTypes.HGV),
-      /*  8 */ getGrossVehicleWeightsSection(VehicleTypes.HGV),
-      /*  9 */ HgvGrossTrainWeight,
-      /* 10 */ HgvMaxTrainWeight,
-      /* 11 */ getAxleWeightsSection(VehicleTypes.HGV),
-      /* 12 */ getTyresSection(VehicleTypes.HGV),
-      /* 13 */ getDimensionsSection(VehicleTypes.HGV, this.vehicleTechRecord.noOfAxles, this.vehicleTechRecord.dimensions?.axleSpacing),
-      /* 14 */ getDimensionsMinMaxSection(
+      /*  7 */ HgvWeight,
+      /*  8 */ getBodySection(VehicleTypes.HGV),
+      /*  9 */ getTyresSection(VehicleTypes.HGV),
+      /* 10 */ getDimensionsSection(VehicleTypes.HGV, this.vehicleTechRecord.noOfAxles, this.vehicleTechRecord.dimensions?.axleSpacing),
```

Comment:
And here

Reply:
The HGV weights are currently broken but that's the next ticket so may as well leave it, what else is broken?